src/app/components/SearchContainer/SearchContainer.jsx

```diff
@@ -59,7 +60,7 @@ class SearchContainer extends React.Component {
       if (query && query.queries) {
         newQuery = Object.assign({}, newQuery, { queries: JSON.parse(query.queries) });
       }
-      if (query) {
+      if (searchQuery && !deepEqual(newQuery, searchQuery)) {
```

Comment:
On closer inspection I don't actually think this is where the unnecessary `searchPost` action is coming from! I'm trying to figure it out, but I think one of the reducers is picking it up and executing it? Can demonstrate what I've found so far.

Reply:
This seems to have been the result of a flubbed merge by me. Can't reproduce after resetting to `HEAD`. Will close this comment.